README.md

```diff
@@ -144,8 +194,18 @@ Added 3 variables.
 nix-repl>
 ```
 
-Then you can tab complete to see what is in `pkgs`
+Then you can tab complete to see what is in `myHaskellPackages`
 
 ```
-nix-repl> pkgs.ex<tab>
+nix-repl> myHaskellPackages.ex<tab>
 ```
+
+Note: the reason we don't have this attribute added by default is you really only need this for debugging and wouldn't want your continuous integration to build the package set as an output of your project.
```

Comment:
I think CI (assuming it simply runs `nix-build`) wouldn't build the nested attribute sets unless marked with `recurseIntoAttrs`.

Reply:
Unfortunately, if you just call `nix-build` it will build all attributes and link them as `result`, `result-2`, `result-3`, etc. Essentially whatever we put here:

```
  {
    inherit shell;
    inherit exe;
    inherit docker;
   ...
  }
```

Will get built if the user calls `nix-build`. I don't use hydra or hercules, nor do I use nix with jenkins so I'm not sure if this would be an issue.